docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Function Worker(Fx)
```

Comment:
Why is this in extensions? Should it be higher up in the cep folder? These seem like a pretty big deal.

Reply:
@diana-macrometa - yes, it is should be a useful feature. But we also need to structure it. And the best place is extensions because it uses another service - FaaS. Also, it must be in execution because it executes some function.